README.md

```diff
@@ -0,0 +1,55 @@
+#########  GOOGLE CLONE ############
+
+Installation
+=======================
+
+1. Clone the repository
+2. Navigate into the server folder
+3. Run npm install to download dependencies
+
+Usage
+=======================
+
+A. Start The backend server
+1. Open a new terminal instance 
+2. Navigate into the server folder
+3. Run nodemon app.js (if installed)
```

Comment:
I didn't have nodemon installed and it as default in the packaje.json
We think that because of this reason it didn't work when we tried to run the server...
```
At the end we added : 
"scripts": {
   "start": "node app.js"
 }
```
After this we did npm start and we were able to see the server side with list of objects.